pyproject.toml

```diff
@@ -83,4 +83,4 @@ requires = ["setuptools>=61.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools.dynamic]
-version = {attr = "aperturedb.__version__"}
```

Comment:
I don't know what added this - I did a `pip install -e` for testing. No idea if we want this or not.

Reply:
We do want this.